Exam/CT/src/exercice1/transform.java

```diff
@@ -0,0 +1,29 @@
+package exercice1;
+public class transform {
+
+public static final int[][] transform(int[][] a) {
+//cr�ation de la classe
+	int[][] matrice = {{0,0,0,0,0},{0, 0 ,0 ,0 ,0},{0, 0 ,360, 0 ,0},{0, 0 ,0 ,0 ,0},{0, 0 ,0 ,0, 0}};
+	
+	int[][] result = null;
+	//double boucle pour matrice
+	for (int i = 0; i< matrice.length;i++) 
+	{
+		for (int j = 0; j<matrice[0].length;j++) 
+		{
+			if(i>=1&&i<=matrice[0].length-2 && j>=1 && j<=matrice.length-2) 
```

Comment:
`i` varies between `0` and `matrice.length`.
`j` varies between `0` and `matrice[0].length`.